README.md

```diff
@@ -75,6 +75,7 @@ commands:
                               [storage <filename>]
                               [ring-buffer <size-in-MB-units>]
                               [block-size <size-in-Byte-units>]
+                              [io-timeout <N-in-Second>]
```

Comment:
@lxbsz This looks nice now. Do you want to add the defaults value at line 81 and may be in the man page too ?

Reply:
Done.